data/data/aws/main.tf

```diff
@@ -1,65 +1,65 @@
 locals {
-  private_endpoints = "${var.tectonic_aws_endpoints == "public" ? false : true}"
-  public_endpoints  = "${var.tectonic_aws_endpoints == "private" ? false : true}"
-  private_zone_id   = "${var.tectonic_aws_external_private_zone != "" ? var.tectonic_aws_external_private_zone : join("", aws_route53_zone.tectonic_int.*.zone_id)}"
+  private_endpoints = "${var.openshift_aws_endpoints == "public" ? false : true}"
+  public_endpoints  = "${var.openshift_aws_endpoints == "private" ? false : true}"
```

Comment:
Besides `tectonic`/`openshift` being redundant.  `aws` is also redundant (it's implied by the module name).  I'd be fine with just `ingress` or some such.

Reply:
> This is a side effect of inlining all of those fields in `pkg/tfvars/config`.

I think Terraform needs a single level.  But we only ever need one platform at a time.  We could (future work?) invert the structs to embed the generic struct within each per-platform struct.